python/util/shipping_companies.py

```diff
@@ -1,4 +1,5 @@
 # when adding new company, make sure last elem in list is custom
 SHIPPING_COMPANIES = ['DHL', 'UPS', 'EMS', 'FedEx',
                       'Australia Post', 'StarTrack', 'Toll', 'TNT',
-                      'Custom Track Link']
+                      'Custom Track Link', 'Fastway',
+                      'realtime express', 'Uber']
```

Comment:
interesting that realtime express is the only one in lower case

Reply:
Yeah. That's how they show it on their site and in their branding, so I thought the representation in our sdk should follow suit.